### PR TITLE
Add Dreamscapes and Dreamscapes 2

### PR DESCRIPTION
**Pull request by Starcommander**
### Description
I added install scripts for 2 new Games
### What works
Installing and playing on Linux and MacOs
### What was not tested
I modified the scripts to match the new syntax from master branch.
That modification was not tested, i used Star_Craft2 as blueprint
### Test
- Operating system: 
Debian10 and 11 via flatpak
Mac Os X 10.13.6 via dmg package
- Hardware (GPU/CPU):
On Debain11 I use a Radeon 5500 XT, that is not supported on flatpak --> software-rendering.

### Ready for review
- [x] Script tested as a regular phoenicis user and working (if you have a problem -> create as draft and ask for help).
- [ ] `json-align` and `eslint` run according to the [documentation](https://phoenicisorg.github.io/scripts/General/tools/). 
- [ ] Codacy and travis checked.

### CODE REVIEW

File: Applications/Games/Dreamscapes2/Online/script.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "Online",
+    "id"                         : "applications.games.dreamscapes2",
+    "compatibleOperatingSystems" : [
+        "MACOSX",
+        "LINUX"
+    ],
+    "testingOperatingSystems" : [],
+    "free"                    : false,
```
Comment (plata) [inline Applications/Games/Dreamscapes2/Online/script.json:9]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes/Online/script.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "Online",
+    "id"                         : "applications.games.dreamscapes",
+    "compatibleOperatingSystems" : [
```
Comment (plata) [inline Applications/Games/Dreamscapes/Online/script.json:4]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes/application.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+    "name"        : "Dreamscapes",
+    "id"          : "applications.games.dreamscapes",
```
Comment (plata) [inline Applications/Games/Dreamscapes/application.json:3]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes/Online/script.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+        "MACOSX",
+        "LINUX"
+    ],
```
Comment (plata) [inline Applications/Games/Dreamscapes/Online/script.json:7]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes2/application.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+    "name"        : "Dreamscapes2",
```
Comment (plata) [inline Applications/Games/Dreamscapes2/application.json:2]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes2/Online/script.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "Online",
```
Comment (plata) [inline Applications/Games/Dreamscapes2/Online/script.json:2]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes2/Online/script.json [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+    "id"                         : "applications.games.dreamscapes2",
+    "compatibleOperatingSystems" : [
+        "MACOSX",
+        "LINUX"
+    ],
+    "testingOperatingSystems" : [],
```
Comment (plata) [inline Applications/Games/Dreamscapes2/Online/script.json:8]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes2/Online/script.json [hunk 8]
```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "Online",
+    "id"                         : "applications.games.dreamscapes2",
+    "compatibleOperatingSystems" : [
```
Comment (plata) [inline Applications/Games/Dreamscapes2/Online/script.json:4]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes2/Online/script.json [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+        "MACOSX",
```
Comment (plata) [inline Applications/Games/Dreamscapes2/Online/script.json:5]: Codacy Issue found: [Format Error: unexpected "    "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes2/application.json [hunk 10]
```diff
@@ -0,0 +1,5 @@
...
+    "id"          : "applications.games.dreamscapes2",
```
Comment (plata) [inline Applications/Games/Dreamscapes2/application.json:3]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes/Online/script.json [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+    "testingOperatingSystems" : [],
+    "free"                    : false,
```
Comment (plata) [inline Applications/Games/Dreamscapes/Online/script.json:9]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes2/Online/script.json [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+        "LINUX"
+    ],
```
Comment (plata) [inline Applications/Games/Dreamscapes2/Online/script.json:7]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes2/application.json [hunk 13]
```diff
@@ -0,0 +1,5 @@
...
+    "description" : "Dreamscapes2: The Sandman is a singleplayer adventure/puzzle game with exciting story. It is hosted on gametop.com"
```
Comment (plata) [inline Applications/Games/Dreamscapes2/application.json:4]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes2/Online/script.json [hunk 14]
```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "Online",
+    "id"                         : "applications.games.dreamscapes2",
+    "compatibleOperatingSystems" : [
+        "MACOSX",
+        "LINUX"
```
Comment (plata) [inline Applications/Games/Dreamscapes2/Online/script.json:6]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes/application.json [hunk 15]
```diff
@@ -0,0 +1,5 @@
...
+    "description" : "Dreamscapes: The Sandman is a singleplayer adventure/puzzle game with exciting story. It is hosted on gametop.com"
```
Comment (plata) [inline Applications/Games/Dreamscapes/application.json:4]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes/Online/script.json [hunk 16]
```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "Online",
+    "id"                         : "applications.games.dreamscapes",
+    "compatibleOperatingSystems" : [
+        "MACOSX",
+        "LINUX"
+    ],
+    "testingOperatingSystems" : [],
```
Comment (plata) [inline Applications/Games/Dreamscapes/Online/script.json:8]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes/application.json [hunk 17]
```diff
@@ -0,0 +1,5 @@
+{
+    "name"        : "Dreamscapes",
```
Comment (plata) [inline Applications/Games/Dreamscapes/application.json:2]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes2/Online/script.json [hunk 18]
```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "Online",
+    "id"                         : "applications.games.dreamscapes2",
```
Comment (plata) [inline Applications/Games/Dreamscapes2/Online/script.json:3]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes/Online/script.json [hunk 19]
```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "Online",
```
Comment (plata) [inline Applications/Games/Dreamscapes/Online/script.json:2]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes/Online/script.json [hunk 20]
```diff
@@ -0,0 +1,11 @@
...
+    "id"                         : "applications.games.dreamscapes",
+    "compatibleOperatingSystems" : [
+        "MACOSX",
+        "LINUX"
```
Comment (plata) [inline Applications/Games/Dreamscapes/Online/script.json:6]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes/Online/script.json [hunk 21]
```diff
@@ -0,0 +1,11 @@
...
+    ],
+    "testingOperatingSystems" : [],
+    "free"                    : false,
+    "requiresPatch"           : false
```
Comment (plata) [inline Applications/Games/Dreamscapes/Online/script.json:10]: Codacy Issue found: [Format Error: unexpected " "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes/Online/script.json [hunk 22]
```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "Online",
+    "id"                         : "applications.games.dreamscapes",
+    "compatibleOperatingSystems" : [
+        "MACOSX",
```
Comment (plata) [inline Applications/Games/Dreamscapes/Online/script.json:5]: Codacy Issue found: [Format Error: unexpected "    "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes/Online/script.json [hunk 23]
```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "Online",
+    "id"                         : "applications.games.dreamscapes",
```
Comment (plata) [inline Applications/Games/Dreamscapes/Online/script.json:3]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

File: Applications/Games/Dreamscapes2/Online/script.json [hunk 24]
```diff
@@ -0,0 +1,11 @@
...
+    "compatibleOperatingSystems" : [
+        "MACOSX",
+        "LINUX"
+    ],
+    "testingOperatingSystems" : [],
+    "free"                    : false,
+    "requiresPatch"           : false
```
Comment (plata) [inline Applications/Games/Dreamscapes2/Online/script.json:10]: Codacy Issue found: [Format Error: unexpected " "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=5674543)

**Comment by plata** [comment]
Please fix the Codacy issues and test the scripts before this can be merged.

**Comment by Starcommander** [comment]
@plata Hello! I fear, there is nothing wrong with the json file.
In my opinion this is a "false positive"
Can you give me any hint, what this "Codacy issues" are complaining about?

The syntax is the same as on "Star Craft 2".
And what is that message "Format error: unexpected "" "??

Thanks in advance.

**Comment by plata** [comment]
This is indeed very strange. I've never seen such a behavior before. I've contacted Codacy to clarify.

**Comment by plata** [comment]
This should be: https://github.com/codacy/codacy-meta/issues/415

**Comment by Zemogiter** [comment]
Shouldn't it be one application per PR?

**Comment by plata** [comment]
Would be better, yes.

**Comment by Starcommander** [comment]
Do I have to split it?
Or can you merge it at once?

**Comment by plata** [comment]
Normally one pull request per app. As the apps belong together, I would be ok with having an exception here. What's more important is that the scripts should be tested.

Apart from that: I've used this PR to test a work around for this strange Codacy behavior. Please merge upstream master to your fork such that the diff is shown correctly.

**Comment by Starcommander** [comment]
I am not able to test any upstream-script because of https://github.com/PhoenicisOrg/phoenicis/issues/2230
Is this a bug?

**Comment by Starcommander** [comment]
@plata I was now able to test both scripts successfully.
Installs and runs with no issue.
But I had to change the freedesktop-version back as described in https://github.com/PhoenicisOrg/phoenicis/issues/2230

File: Applications/Games/Dreamscapes/Online/script.json (hunk 11 above)
Comment (plata) [inline Applications/Games/Dreamscapes/Online/script.json:9]: The URL is `free-games-download`: Is it free or not?

File: Applications/Games/Dreamscapes2/Online/script.json (hunk 1 above)
Comment (plata) [inline Applications/Games/Dreamscapes2/Online/script.json:9]: See above

File: Applications/Games/Dreamscapes/Online/script.json (hunk 11 above)
Comment (Starcommander) [reply Applications/Games/Dreamscapes/Online/script.json:9]: It is free of costs.
But it is not open source.
I will change it to true.
But I also suggest to distinguish between "free of cost" and free (regarding license).
So adding an additional field in future would be helpful, imho.

**Comment by Starcommander** [comment]
Personally I do not like spaces in file names and directories.
But it is your decision how it matches the rules of POL.
... in progress ...

**Comment by Starcommander** [comment]
Adding space and change preferences to free: done.
I also tested the script with "Dreamscapes 2" and it works.

File: Applications/Games/Dreamscapes/Online/script.json (hunk 11 above)
Comment (plata) [reply Applications/Games/Dreamscapes/Online/script.json]: Feel free to open a new issue to discuss this. Currently, it's free as in beer (see [docs](https://phoenicisorg.github.io/scripts/Develop/script-json/)).

**Pull request merged**